invenio_github/api.py

```diff
@@ -623,6 +623,9 @@ def test_zipball(self):
             response.status_code == 200
         ), f"Could not retrieve archive from GitHub: {zipball_url}"
 
+        # Overwrite the original URL with the newly found more specific URL.
+        self.release_payload["zipball_url"] = zipball_url
```

Comment:
just out of curiosity, is there any reason why it is not assigned before the assert (622)?

Reply:
No particular reason, I was just thinking "if the status code is 200, then the URL is OK and can be stored".
But the assert would stop the code execution flow anyway, so it doesn't matter.